### DISCUSSION TITLE
Understanding position argument in Transmitter/Receiver function in ray tracing Module

### DISCUSSION THREAD

**Discussion by rietesh** [NVlabs/sionna#154] [General]
I was going through the tutorial [here](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#). When initializing the transmitter and receiver, we set position and orientation; I'm having a hard time understanding how the position values are set. What coordinate system do we use? Can I use lat/long values? That does not make sense. 

I have the lat/long values of transmitters, but after importing the XML file from mitsuba blender, I think the lat/long values are not preserved. How can I go about using lat/long values to initialize transmitters?

**Comment by jhoydis** [comment]
Hi,

The positions of the transmitters and receivers are in the Cartesian coordinates.

**Reply by rietesh** [reply]
Hi, @jhoydis 

Thank you for your quick reply Is there a way to plot the coordinates along with building outlines? I see we can use `scene.render` and bird's view angle to achieve that, but the output figure does not have x and y-axis labels, is there any way we can render it on the cartesian coordinates?

**Reply by jhoydis** [reply]
No, you can't for the moment. But you can use the (x,y,z) coordinates from Blender.

**Reply by rietesh** [reply]
@jhoydis Are the coordinates from blender the same in the Sionna RT module?

**Reply by jhoydis** [reply]
Yes.

**Reply by rietesh** [reply]
@jhoydis I just verified both the coordinate systems, and they do not match i.e, the Globad coordinate system of Blender and the position coordinate system in sionna do not match

**Reply by faycalaa** [reply]
Hi,

When importing/exporting a scene in Blender with the Mitsuba (Sionna) file format, it's important to set Z-axis as "up" and Y-axis as "forward". This should align the coordinates correctly.

**Reply by rietesh** [reply]
@faycalaa Yes I did that correctly: 
 Sionna origin(0,0) [here](https://imgur.com/yDMvVli)
 Blender Origin [here](https://imgur.com/X2qYifU)

**Reply by jhoydis** [reply]
In all of our example scenes, the coordinate systems match. In order to help you, we would need access to your scene in Blender to see what goes wrong.

**Reply by rietesh** [reply]
Hi @jhoydis Sure, here is my .blend [file](https://drive.google.com/file/d/1tz5CmIFCAfqC53seLO8KTPPphsnAJU2G/view?usp=sharing) I use the coordinate system transformation from here https://github.com/vvoovv/blender-osm/wiki/Georeferenced-Blender-Scene 

When I match the position calculated from the above link, it does not match after importing it into Sionna. I use Z up and Y forward while exporting.

**Reply by merlinND** [reply]
Hello @rietesh,

I tried reproducing the mismatch of coordinate systems by adding a sphere in Blender in your scene at [0, 0, 0], exporting the scene to the Mitsuba format, loading it in Sionna, and then adding a Sionna receiver at [0, 0, 5]. As you can see, they are perfectly aligned over `x = 0, y = 0`:

![image](https://github.com/NVlabs/sionna/assets/5563370/2a9bd6fc-ccb3-4731-a4f4-33ab1e76215e)

Are you by any chance running into the issue below? (although it usually manifests in much more broken geometry)
If you aren't already, I recommend using Blender 3.3 until the fix is merged to the mitsuba-blender plugin.
https://github.com/mitsuba-renderer/mitsuba-blender/issues/58

**Comment by rietesh** [comment]
@jhoydis @faycalaa 

What is the transmitter power used in Sionna, or the transmitter voltage in creating coverage maps?

**Reply by jhoydis** [reply]
The power is one. So you can think about a coverage map as a pathloss map.
